website/templates/project/project_base.mako

```diff
@@ -32,6 +32,7 @@
 ##  % endif
 % endif
 
+<%include file="modal_mailing_list_contributors.mako"/>
```

Comment:
The mailing list modal does not appear to be used on every project page. Putting it in the project_base template may add bloat to pages that don't (and can't) display it. I suggest adding it to the project dashboard and settings pages specifically.